package.json

```diff
@@ -68,17 +68,11 @@
       }
     },
     "snap": {
-      "synopsis": "Unofficial Microsoft Teams client for Linux using Electron. It uses the Web App and wraps it as a standalone application using Electron.",
+      "synopsis": "teams-for-linux",
+      "description": "Unofficial Microsoft Teams client for Linux using Electron. It uses the Web App and wraps it as a standalone application using Electron",
+      "summary": "teams-for-linux",
       "confinement": "strict",
-      "grade": "stable",
-      "buildPackages": [],
-      "stagePackages": [
-        "default"
-      ],
-      "plugs": [
-        "default",
-        "camera"
-      ]
+      "grade": "stable"
```

Comment:
The plug part is missing, this provides camera access.
```
      "plugs": [
        "default",
        "camera"
      ]
```

Reply:
Fixed in 7ac87eeb5d5adeb4f5f8d8512ae19bce749492a0